app/API/V1/Entities/User.php

```diff
@@ -530,6 +535,8 @@ public function getTTConfig(): array
             'admin'=>[ // admins can do the same thing as users except to any user, and they do not have update and delete restricted
                 'create'=>[
                     'extends'=>[':user:create'],
+                    'allowed'=>true,
+                    'permissive'=>true,
```

Comment:
This gives admin the ability to update roles, permissions, and passwords. I think that should probably be something only super admin can do. Permissive means that unless you tell them they can't do something, they can do everything. Part of the problem with putting this on admin it is means admins can make them selves super admins.